src/components/basket-item/basket-item.jsx

```diff
@@ -4,27 +4,69 @@ var React = require('react');
 module.exports = React.createClass({
   propTypes: {
     currencySymbol: React.PropTypes.string,
+    freeText: React.PropTypes.string,
+    removeText: React.PropTypes.string,
     price: React.PropTypes.number,
+    title: React.PropTypes.node,
+    toggleDescription: React.PropTypes.bool,
```

Comment:
`toggle` implies an action, and this is a state: can we rename this property to something else like `isDescriptionVisible` or whatever fits better with the naming conventions but implies state?